pkg/commands/flagsets.go

```diff
@@ -0,0 +1,100 @@
+}
+
+func setupOutputFlagSet(v *viper.Viper, fs *pflag.FlagSet) {
+	internal.Vibra(v, fs, fs.String, "out-format", "output.format", config.OutFormatColoredLineNumber,
+		wh(fmt.Sprintf("Format of output: %s", strings.Join(config.OutFormats, "|"))))
+	internal.Vibra(v, fs, fs.Bool, "print-issued-lines", "output.print-issued-lines", true, wh("Print lines of code with issue"))
+	internal.Vibra(v, fs, fs.Bool, "print-linter-name", "output.print-linter-name", true, wh("Print linter name in issue line"))
+	internal.Vibra(v, fs, fs.Bool, "uniq-by-line", "output.uniq-by-line", true, wh("Make issues output unique by line"))
+	internal.Vibra(v, fs, fs.Bool, "sort-results", "output.sort-results", false, wh("Sort linter results"))
+	internal.Vibra(v, fs, fs.String, "path-prefix", "output.path-prefix", "", wh("Path prefix to add to output"))
+}
+
+//nolint:gomnd
+func setupIssuesFlagSet(v *viper.Viper, fs *pflag.FlagSet) {
+	fs.StringSliceP("exclude", "e", nil, wh("Exclude issue by regexp")) // Hack see Loader.applyStringSliceHack
+	internal.Vibra(v, fs, fs.Bool, "exclude-use-default", "issues.exclude-use-default", true, getDefaultIssueExcludeHelp())
+	internal.Vibra(v, fs, fs.Bool, "exclude-case-sensitive", "issues.exclude-case-sensitive", false,
+		wh("If set to true exclude and exclude rules regular expressions are case-sensitive"))
+
+	internal.Vibra(v, fs, fs.Int, "max-issues-per-linter", "issues.max-issues-per-linter", 50,
+		wh("Maximum issues count per one linter. Set to 0 to disable"))
+	internal.Vibra(v, fs, fs.Int, "max-same-issues", "issues.max-same-issues", 3,
+		wh("Maximum count of issues with the same text. Set to 0 to disable"))
+
+	const newDesc = "Show only new issues: if there are unstaged changes or untracked files, only those changes " +
+		"are analyzed, else only changes in HEAD~ are analyzed.\nIt's a super-useful option for integration " +
+		"of golangci-lint into existing large codebase.\nIt's not practical to fix all existing issues at " +
+		"the moment of integration: much better to not allow issues in new code.\nFor CI setups, prefer " +
+		"--new-from-rev=HEAD~, as --new can skip linting the current patch if any scripts generate " +
+		"unstaged files before golangci-lint runs."
+	internal.VibraP(v, fs, fs.BoolP, "new", "n", "issues.new", false, wh(newDesc))
+	internal.Vibra(v, fs, fs.String, "new-from-rev", "issues.new-from-rev", "",
+		wh("Show only new issues created after git revision `REV`"))
+	internal.Vibra(v, fs, fs.String, "new-from-patch", "issues.new-from-patch", "",
+		wh("Show only new issues created in git patch with file path `PATH`"))
+	internal.Vibra(v, fs, fs.Bool, "whole-files", "issues.whole-files", false,
+		wh("Show issues in any part of update files (requires new-from-rev or new-from-patch)"))
+	internal.Vibra(v, fs, fs.Bool, "fix", "issues.fix", false, wh("Fix found issues (if it's supported by the linter)"))
+}
+
+func wh(text string) string {
```

Comment:
Maybe take the opportunity to improve the naming, `wh` doesn't give much info without context.